tools/mutator-utils/mutator_helper.cpp

```diff
@@ -1,5 +1,7 @@
 #include "mutator_helper.h"
 #include "mutator.h"
+#include <iostream>
+#include <random>
```

Comment:
I don't think we need both `include` here?
random is included in `util.cpp`, and I don't see why we need an `iostream` here